packages/python/m/core/rw.py

```diff
@@ -28,18 +28,19 @@ def read_file(filename: str | None) -> OneOf[Issue, str]:
         )
 
 
-def write_file(filename: str, text: str) -> OneOf[Issue, int]:
+def write_file(filename: str, text: str, mode: str = 'w') -> OneOf[Issue, int]:
     """FP version of open to write to a file.
```

Comment:
```suggestion
    """FilePath version of open to write to a file.
```

unless we're talking about this bad boy
![The_FP](https://github.com/jmlopez-rod/m/assets/62945622/876f5390-b536-4e84-8b5e-9c959fd13695)

Reply:
Rewrote 
![Screenshot from 2023-11-08 21-10-32](https://github.com/jmlopez-rod/m/assets/1810397/d6ccb990-f44f-46e8-9762-991128b00cf3)

An attempt was made.